sources/common/dbdump.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
```

Comment:
Packages should have a comment of the form "Package ABC ..." e.g. internal/progress.go (as outlined in https://golang.org/doc/effective_go#commentary).

Basically, exactly one file in a package should have a package comment.

We seem to have gotten away from this in some recent PRs.